index.html

```diff
@@ -355,6 +359,14 @@ <h2><code>PointerEvent</code> Interface</h2>
                         <dd>
                             <p>The clockwise rotation (in degrees, in the range of [0,359]) of a transducer (e.g. pen stylus) around its own major axis. For hardware and platforms that do not report twist, the value MUST be 0.</p>
                         </dd>
+                    <dt><dfn>altitudeAngle</dfn></dt>
+                        <dd>
+                            <p>The altitude (in radians) of the transducer (e.g. pen stylus), in the range [0,π/2] - where 0 is parallel to the X-Z plane, and π/2 is perpendicular to the X-Z plane. The value 0 should be used for devices which do not support this property.</p>
```

Comment:
Based on https://w3c.github.io/touch-events/#dom-touch-altitudeangle I believe the plane should be X-Y not X-Z. When reading the definitions I found the touch-events one more intuitive: pen parallel to the surface / perpendicular to the surface.

Reply:
Agree, the X-Z thing was my mistake. I've corrected this and expanded this a bit to talk about the surface.